package.json

```diff
@@ -227,7 +227,7 @@
     "resin-compose-parse": "^2.1.0",
     "resin-doodles": "^0.1.1",
     "resin-image-fs": "^5.0.8",
-    "resin-multibuild": "^4.3.4",
+    "resin-multibuild": "^4.4.1-update-deps-9aede6efa97b9da634e42cc6f11c46b81ff5c077",
```

Comment:
Depends on https://github.com/balena-io-modules/resin-multibuild/pull/72 for unpinning dockerode